en/components/expansion_panel.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Expansion Panel
+_description: Provides an easily configurable component with two states - collapsed and expanded.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Collapsible component, IgxExpansionPanel, Ignite UI Expansion Panel
+---
+
+## Expansion Panel
+<p class="highlight">
+The `IgxExpansionPanel` is a lightweigh component which can be rendered in two states - collapsed or expanded. When collapsed, the panel displays a short summary of its content. The collapsed panel can be toggled (via click or keyboard interactions) to display the whole body of the component, containing any additional necessary content.
```

Comment:
Let's reword second sentence to The collapsed panel can be toggled using mouse click, or keyboard interactions (maybe have link here) to display the body completely, containing....